solidity/src/vaults/OneWayVault.sol

```diff
@@ -128,7 +128,10 @@ contract OneWayVault is
      * @param depositAccount Account where deposits are held
      * @param strategist Address of the vault strategist
      * @param depositFeeBps Fee charged on deposits in basis points (1 BPS = 0.01%)
-     * @param withdrawRateBps Fee charged on withdrawals in basis points (1 BPS = 0.01%)
+     * @param withdrawFeeBps Fee charged on withdrawals in basis points (1 BPS = 0.01%)
+     * @param maxRateIncrementBps Maximum allowed increase in redemption rate per update (in basis points)
```

Comment:
Can we be super clear that this is *relative* increase/decrease as opposed to an absolute increase/decrease? Suggestion:
```solidity
* @param maxRateIncrementBps Maximum allowed relative increase in redemption rate per update (in basis points). For example, if maxRateIncrementBps is 100 (1%), at a redemption rate of 2.00, the new rate can go up to 2.02.
* @param maxRateDecrementBps Maximum allowed relative decrease in redemption rate per update (in basis points). For example, if maxRateDecrementBps is 50 (0.5%), at a redemption rate of 2.00, the new rate can go down to 1.99
 ```